### ISSUE TITLE
CTP scan - imageIds order is not calculated correctly

### ISSUE THREAD

**Issue by sharon-shrits** [cornerstonejs/cornerstone3D-beta#555]
When opening a CTP series which has multiple time points (few scans in one series), and there is more than 1 slice with the same z position, the order of the imageIds (sortImageIdsAndGetSpacing) is calculated wrong when using `wadors`.

The expected result is that the order of the multiple scans will remain the same (displayed one after the other),
but the actual result is that all of the dicoms which have the same z position, are displayed one after another instead of its original scan position.

example from using OHIF-Viewer (wrong ordering):
https://user-images.githubusercontent.com/92016239/232318239-f6fe8fd8-d46c-4da8-801e-8e117ce8691c.mov

example from using Horos (correct ordering):
https://user-images.githubusercontent.com/92016239/232319445-210f25b8-6292-42eb-a01e-2db35d1ec0e4.mp4

Would appreciate checking it, Thank you!

**Comment by sedghi** [comment]
Hmmm interesting. Can you show me how does Horos handle the MPR for such series? Can you try to load it as Sagittal or Coronal?

**Comment by sharon-shrits** [comment]
> Hmmm interesting. Can you show me how does Horos handle the MPR for such series? Can you try to load it as Sagittal or Coronal?

@sedghi 

https://user-images.githubusercontent.com/92016239/232786668-205e0554-72b6-4635-9bc7-da0756af8717.mp4

**Comment by sedghi** [comment]
But the bottom one seems wrong, am i seeing it correctly? it has some artifacts in the middle of the head

**Comment by sedghi** [comment]
Yeah it is volumeViewport. There might be a bug in the calculation of the position, not sure yet. Also, sharing anonymized data always helps
